src/Components/Activities/ActivitiesForm.js

```diff
@@ -9,7 +9,7 @@ import { Alert } from '@mui/material';
 import { CKEditor } from '@ckeditor/ckeditor5-react';
 import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
 import { useDropzone } from 'react-dropzone';
-import { dropzoneConfig, isEmptyList } from '../../Utils/ActivitiesUtils';
+import { dropzoneConfig, isEmptyList } from '../../Utils';
```

Comment:
are this functions on Utils?

Reply:
it looks like isEmptyList is not

Reply:
yes, they are, both methods are in Utils/index.js

Reply:
This is the actual branch screenshot

![image](https://user-images.githubusercontent.com/59523415/146070527-1d2ea9fc-8051-4006-a20a-97d2d33dd431.png)